.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,14 @@
+name: Deploy to Staging
+on:
+  push:
+    branches: [ 'master' ]
```

Comment:
should we also deploy from staging branch? so you can test things before you merge to master 🤔

Reply:
Agreed. @davidhanh @aashishgurung. WDYT?

Reply:
@davidhanh suggest to go with this new flow [Branching Strategy](https://opn-ooo.atlassian.net/wiki/spaces/D/pages/576754440/Branching+Strategy+-+Scaled+Trunk+Based+Development).

cc: @kan98 @aashishgurung

Reply:
@kan98 @ajzkk Once our PR is merged and deployed to the staging, QA will do the testing in the staging and reopens the ticket if they find any issue. So, I don't think we need staging branch. Plus current process complies with the new branching strategy.

Reply:
yea that's up to you guys how it'll work best for you. I agree that if you're always gonna test on master first then this is fine. I'll approve the PR